mobile/src/components/common/button/button.tsx

```diff
@@ -11,14 +18,30 @@ type Props = {
   icon?: IconName;
   variant?: ButtonVariant;
   onPress: () => void;
+  style?: ViewStyle;
+  isLoading?: boolean;
+  loaderColor?: string;
```

Comment:
I've passed styles to have an opportunity change button's vertical padding, because common button is too high for this view
Also, I've added loader color, because by default its blue like the Primary button background, but if we will use this loader only for primary buttons, I'll set it white by default

Reply:
Ok, I like your first option in this comment https://github.com/BinaryStudioAcademy/bsa-2022-guruhub/pull/368#issuecomment-1235412762

Reply:
I think we can set loader color depending on button variant

Reply:
Also about styles, it's not a good idea to have such customization, can't we avoid passing styles there? I think it's better to add button size property with type 'small' | 'large'